pkg/services/guardian/guardian_test.go

```diff
@@ -726,3 +726,45 @@ func TestGuardianGetHiddenACL(t *testing.T) {
 		})
 	})
 }
+
+func TestGuardianGetAclWithoutDuplicates(t *testing.T) {
+	Convey("Get hidden ACL tests", t, func() {
```

Comment:
Since you're adding a new test, can it be done without go-convey?

Reply:
Good catch. Fixed